client/lib/two-step-authorization/index.js

```diff
@@ -35,6 +35,7 @@ function TwoStepAuthorization() {
 	this.smsResendThrottled = false;
 	this.bumpMCStat = function( eventAction ) {
 		analytics.mc.bumpStat( '2fa', eventAction );
+		analytics.tracks.recordEvent( 'calypso_two_step', { event_action: eventAction } );
```

Comment:
A final not about this event name. The template is:

> `<source>_<context>_<optional-subcontext>_<action>_<optional-qualifier>`

Something like `calypso_login_twostep_authorize` might adhere better.